sdk/go/common/resource/properties_path_test.go

```diff
@@ -252,6 +252,22 @@ func TestPropertyPath(t *testing.T) {
 			}
 		})
 	}
+
+	negativeCasesStrict := []string{
+		// Syntax erros
+		`root.array.[1]`,
+		`root.["key with a ."]`,
+	}
+	//nolint:paralleltest
```

Comment:
```suggestion
	//nolint:paralleltest // false positive because range var isn't used directly in t.Run(name) arg
```

Similar to above, it's helpful to have this information to quickly understand why the linter is turned off.